omero_figure/templates/figure/index.html

```diff
@@ -581,14 +581,19 @@ <h4>Dynamic properties</h4>
                             <td><i>Height</i>: <strong>300</strong></td>
                         </tr>
                         <tr>
-                            <th>Time with decimal precision of 2</th>
-                            <td>Time (s): [time.seconds.2]</td>
-                            <td>Time (s): 0.93 s</td>
+                            <th>Width with decimal precision of 3</th>
+                            <td>Width: [width.unit.3]</td>
+                            <td>Width: 4.450 µm</td>
                         </tr>
                         <tr>
                             <th>Time with frame n°3 as the "zero"</th>
```

Comment:
I'm not sure that "zero" is the best way to explain this? Maybe "Label shows time relative to the 3rd frame". Or "Time difference between current and 3rd frames"?

Reply:
"Time relative to the 3rd frame" sounds great